src/zenml/constants.py

```diff
@@ -256,3 +256,36 @@ def handle_int_env_var(var: str, default: int = 0) -> int:
 # Service connector constants
 DOCKER_REGISTRY_RESOURCE_TYPE = "docker-registry"
 KUBERNETES_CLUSTER_RESOURCE_TYPE = "kubernetes-cluster"
+
+# Stack Recipe constants
+STACK_RECIPE_PACKAGE_NAME = "mlstacks"
+STACK_RECIPE_TERRAFORM_FILES_PATH = "terraform"
+STACK_RECIPES_GITHUB_REPO = "https://github.com/zenml-io/mlops-stacks.git"
+STACK_RECIPES_REPO_DIR = "zenml_stack_recipes"
+ALPHA_MESSAGE = (
+    "The mlstacks tool/package is in alpha and actively being developed. "
+    "Please avoid running mission-critical workloads on resources deployed "
+    "through these commands. If you encounter any problems, create an issue "
+    f"on the repository {STACK_RECIPES_GITHUB_REPO} and we'll help you out!"
+)
+NOT_INSTALLED_MESSAGE = (
+    "The prerequisites for using `mlstacks` (the `mlstacks` and "
+    "`python-terraform` packages seem to be unavailable on your machine "
+    "and/or in your environment. To install the missing dependencies: \n\n"
+    "`pip install mlstacks`"
+)
+TERRAFORM_NOT_INSTALLED_MESSAGE = (
+    "Terraform appears not to be installed on your machine and/or in your "
+    "environment. Please install Terraform and try again."
+)
+STACK_RECIPE_MODULAR_RECIPES = ["aws", "gcp", "k3d"]
+MLSTACKS_SUPPORTED_STACK_COMPONENTS = [
+    "artifact_store",
+    "container_registry",
+    "experiment_tracker",
+    "orchestrator",
+    "model_deployer",
+    "model_registry",
+    "mlops_platform",
+    "step_operator",
+]
```

Comment:
(IMO we should do the same for server endpoints/... but that's another story I guess)

Reply:
It's only 30 lines of `mlstacks` constants and the `mlstacks` modules has since been deleted. I'd rather keep it here for now until we really start breaking it up.